bower.json

```diff
@@ -57,7 +57,8 @@
     "DOMPurify": "0.8.3",
     "jquery-tiny-pubsub": "https://github.com/cowboy/jquery-tiny-pubsub.git#~0.7.0",
     "markdown-it": "7.0.1",
-    "bootstrap-switch": "3.3.2"
+    "bootstrap-switch": "3.3.2",
+    "At.js": "~1.5.2"
```

Comment:
@orangejenny you mentioned pulling this out of HQ because it sometimes causes issues with a jquery plugin we use. Do you think this usage is safe?